compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -529,3 +571,249 @@ def _process_obs(self, curr_obs, sequence_example):
           curr_obs_feature, dtype=obs_dtype, name=curr_obs_feature_name)
       add_feature_list(sequence_example, curr_obs_feature,
                        curr_obs_feature_name)
+
+
+class ModuleWorkerUtility:
```

Comment:
Nit: Names like "utility" or "helper" never convey much information. How about instead `ModuleWorkerResultProcessor` or `Filter` or `Saver`? (or maybe there's a better name)?

Reply:
Changed the name.